tools/memap.py

```diff
@@ -9,6 +9,7 @@
 import json
 import argparse
 from prettytable import PrettyTable
+from StringIO import StringIO
```

Comment:
Think we should rather reuse what we have and not add new unique python class that will be included to every build, export, etc? Let's use what we have here @bridadan and minimize dependencies :)